nixos/modules/services/networking/prosody.nix

```
@@ -418,15 +427,28 @@ let
       httpUploadPath = mkOption {
         type = types.str;
         description = lib.mdDoc ''
-          Directory where the uploaded files will be stored. By
-          default, uploaded files are put in a sub-directory of the
+          Directory where the uploaded files will be stored
+          when the http_upload module is used.
+          By default, uploaded files are put in a sub-directory of the
           default Prosody storage path (usually /var/lib/prosody).
```

```suggestion
          Directory where the uploaded files will be stored when the http_upload module is used.
          By default, uploaded files are put in a sub-directory of the default Prosody storage path (usually /var/lib/prosody).
```
we don't have a strict line limit, so we might as well format this to be somewhat readable.